input/fsh/profiles/FiSchedulingLocation.fsh

```diff
@@ -0,0 +1,38 @@
+Profile: FiSchedulingLocation
+Parent: Location
+Id: fi-scheduling-location
+Description: "Details for schedulable location"
+* ^meta.lastUpdated = "2020-01-02T13:46:59.349+00:00"
```

Comment:
We don't need the lastUpdated for the first batch of profiles, right? Also, I would assume the ig authoring toolchain to keep track of these automagically. Not sure, though.